Sopc2DTS.java

```diff
@@ -122,6 +125,7 @@ public Sopc2DTS() {
 		vOptions.add(new CommandLineOption("type", 		"t", outputType, 		true, false,"The type of output to generate", "{dtb,dtb-hex8,dtb-hex32,dtb-char-arr,dts,uboot,kernel}"));
 		vOptions.add(new CommandLineOption("bootargs", 	null,bootargs,	 		true, false,"Default kernel arguments for the \"chosen\" section of the DTS", "kernel-args"));
 		vOptions.add(new CommandLineOption("sopc-parameters", 	null,sopcParameters, true, false,"What sopc-parameters to include in DTS", "{node,cmacro,all}"));
+		vOptions.add(new CommandLineOption("extra-components",	"e", extraComponents, 	true, false,"Extra component descriptions (see source for examples) (can be used many times)", "sopc component xml file"));
 
```

Comment:
Just a nitpick...
Could you rename 'extra-components' to 'extra-componentlibs'?
Here and in the lines below